packages/app/src/components/DBTable/DBRowTableFieldWithPopover.tsx

```diff
@@ -151,7 +151,6 @@ export const DBRowTableFieldWithPopover = ({
         position="top-start"
         offset={5}
         opened={opened}
-        zIndex={1}
```

Comment:
This is great because I can see the popovers now, but I'm also seeing that the add / exclude filter buttons are not having an effect when in the side drawer other than closing the side drawer:

https://github.com/user-attachments/assets/2457431f-daac-43fd-9d17-a32c08f0b173